automated/health-checks/tests/health-check-002.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+TEST_CASE_NAME=$(basename "$0" .sh)
+
+# Check for errors and warnings in dmesg
+if dmesg | grep -E 'error|fail|warning' >/dev/null; then
+    lava-test-case "$TEST_CASE_NAME-Errors or Warnings in dmesg: Found" --result fail
```

Comment:
Can we abbreviate this? Something like "dmesg not clean"? It's hard to read and easily blends in with LAVA's chatter about tests in the logs.